task/gcp/resources/data_source_image.go

```diff
@@ -32,7 +32,7 @@ func (i *Image) Read(ctx context.Context) error {
 	image := i.Identifier
 	images := map[string]string{
 		"ubuntu": "ubuntu@ubuntu-os-cloud/ubuntu-2004-lts",
-		"nvidia": "ubuntu@nvidia-ngc-public/nvidia-gpu-cloud-image-20211105",
+		"nvidia": "ubuntu@deeplearning-platform-release/common-cu113-ubuntu-2004",
```

Comment:
would say cu112 rather than cu113

- mentioned explicitly by https://www.tensorflow.org/install/gpu#software_requirements
- matches AWS DLAMI

https://datascience.stackexchange.com/questions/97849/cuda-for-pytorch-and-cuda-for-tensorflow is for another day :)

Reply:
Better if you ask for something [in the realms of possibility](https://cloud.google.com/deep-learning-vm/docs/images); CUDA 11.3 is compatible with TensorFlow 2.8 as per Google Cloud, anyway.

Bonus point for you, because it's not listed on [the TensorFlow compatibility section](https://www.tensorflow.org/install/source#gpu).

Reply:
ok - in which case probably worth updating the AWS DLAMI to CUDA 11.3 too

Reply:
Feel free to, if you have the machine image string.

Reply:
Resolved with https://github.com/iterative/terraform-provider-iterative/pull/533/commits/ea2f6ff27b3cb9c6a5f0af6de8a40a976f0ed1d4